### DISCUSSION TITLE
Latest Grinding Guide Publication having increased cooldown

### DISCUSSION THREAD

**Discussion by JustDorien** [HelixNebulaStudio/RiseOfTheDead#412] [General]
Hello, alot of rise of the dead players are complaining about the Latest Grinding Guide Publication cooldown being increased, a lot players are mad about it, and I think you should listen to what community mainly wants, and revert the cooldown back to normal. Or at least make it 1h, and not 5 hours which is insane.

**Comment by GregTheBestest** [comment]
The pass has additional rewards at the end, one of possible rewards is gold. As Khronos said, the end pass has a gold pool. With increase of the cooldown he tries to make the gold at the end of the pass more accessible to those who don't grind the game most of their time during day, so everybody gets a chance to get some gold.

**Comment by Leroy8888** [comment]
#418
